versioned_docs/version-8.6/self-managed/concepts/elasticsearch-without-cluster-privileges.md

```diff
@@ -139,6 +154,7 @@ Create a configuration for the Camunda single application with the following val
 zeebe.broker.exporters.elasticsearch:
   className: io.camunda.zeebe.exporter.ElasticsearchExporter
   args:
+    url: https://localhost:9200
```

Comment:
:question: Should we document the workaround required if you use self signed certificate? That is to build your own trust store, and then override the `javax.net.ssl.trustStore` property? AFAIK this is not documented anywhere. Though it should probably be documented on the Elasticsearch exporter page (which I can do :+1:)

Reply:
Sounds reasonable to me. Then we can also easier link to it.